mlir/include/mlir/Dialect/Tosa/Transforms/Passes.td

```diff
@@ -99,7 +89,9 @@ def TosaValidation : Pass<"tosa-validate", "mlir::ModuleOp"> {
   let options = [
       ListOption<"profile", "profile", "std::string",
              "Validate if operations match for the given profile set">,
-      Option<"StrictOperationSpecAlignment", "strict-op-spec-alignment", "bool",
+      ListOption<"extension", "extension", "std::string",
```

Comment:
Does this allow multi-extension setting?

Reply:
Yes they do e.g.
```
--tosa-validate=profile=pro_fp,pro_int extension=int16,bf16,fft,int4
```